src/components/TokenLocking/index.tsx

```diff
@@ -47,6 +47,9 @@ const TokenLocking = () => {
           <PaperContainer>
             <Leaderboard />
           </PaperContainer>
+          <Link href={AppRoutes.terms} component={NextLink} m={2} sx={{ textAlign: 'center' }}>
+            Terms and Conditions
+          </Link>
```

Comment:
In the designs this link does not exist at all anymore. Instead there is a legal disclaimer at the bottom.
We can adjust that in a separate PR though.

Reply:
merging this PR, will update the link separately